_layouts/docwithnav.html

```diff
@@ -12,21 +12,38 @@
 	{% include documentation.html %}
 </div>
 <script type="text/javascript">
-	$(document).ready(function() {
-		/* Find and set target="_blank" for all external URLs (in navigation menu and content body) so that they open in a new tab/window */
-        $('a').filter(function() {
-			return this.hostname && this.hostname !== location.hostname;
-		}).attr('target', '_blank');
-	});
+    $(document).ready(function() {
+		/* On page load, open the navigation menu and highlight the menu item that corresponds to the current URL */
+		var curnavitem = $('#wrapper a[href="' + window.location.href + '"]');
```

Comment:
I confirm this does highlight the current page in the nav bar, so
progress.  LGTM

A few issues still:


It closes and opens menu items on each back-and-forth, even if that
sub-menu was already open.

Sometimes back-button does not refresh the page.

1) load
http://richieescarez.github.io/kubernetes/v1.0/docs/user-guide/secrets.html
2) Click annotations in the left bar
3) click back-button



On Fri, Jul 31, 2015 at 12:17 PM, RichieEscarez <notifications@github.com>
wrote:

> In _layouts/docwithnav.html
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12092#discussion_r36006962>
> :
>
> > @@ -12,21 +12,38 @@
> >  	{% include documentation.html %}
> >  </div>
> >  <script type="text/javascript">
> > -	$(document).ready(function() {
> > -		/* Find and set target="_blank" for all external URLs (in navigation menu and content body) so that they open in a new tab/window */
> > -        $('a').filter(function() {
> > -			return this.hostname && this.hostname !== location.hostname;
> > -		}).attr('target', '_blank');
> > -	});
> > +    $(document).ready(function() {
> > +		/* On page load, open the navigation menu and highlight the menu item that corresponds to the current URL */
> > +		var curnavitem = $('#wrapper a[href="' + window.location.href + '"]');
>
> Change this from window.location.pathname to window.location.href (to find
> absolute URLs instead of just the suffix - result of difference in local
> build config)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/12092/files#r36006962>
> .
>